experiments/rodm/make_plots.jl

```diff
@@ -0,0 +1,633 @@
+### A Pluto.jl notebook ###
+# v0.17.7
+
+using Markdown
```

Comment:
Can you just add a very quick description of this notebook? I am guessing it is to  make all the plots from previously generated data right?